firmware/quintuna/VC/src/io/io_lowVoltageBattery.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include "hw_gpio.h"
+<<<<<<< HEAD
+#include "cmsis_os2.h"
+
+extern osSemaphoreId_t bat_mtr_sem;
+
+extern const uint16_t ACCUMULATED_CHARGE_COMMAND;
+extern const uint16_t CELL0_VOLTAGE_COMMAND;
+extern const uint16_t CELL1_VOLTAGE_COMMAND;
+extern const uint16_t CELL2_VOLTAGE_COMMAND;
+extern const uint16_t CELL4_VOLTAGE_COMMAND;
+extern const uint16_t STACK_VOLTAGE_COMMAND;
```

Comment:
can you move this a bit lower alongwide `io_lowVoltageBattery_get_voltage` which is where I presume this is being used?

Also, can you turn this into an enum which is exposed here?